tests/testutil.h

```diff
@@ -201,12 +201,12 @@ static void NN_UNUSED test_drop_impl (char *file, int line, int sock, int err)
     }
 }
 
-static int get_test_port(int argc, const char *argv[])
+static int NN_UNUSED get_test_port (int argc, const char *argv[])
```

Comment:
Example of precedent: https://github.com/nanomsg/nanomsg/pull/696/files/cdcb73a7dbebfaf9bb98190baef4372df1720937#diff-650a958e586e7f65e22e0174b4bac596R186